src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -3190,6 +3226,19 @@ static void updateEntity
     currentGeoJson = getObjectFieldF(loc, ENT_LOCATION_COORDS);
   }
 
+  /* Is the entity using date expiration? In that case, we fill the currentdateExpiration attribute with that information.
+   * In any case, if the request contains a new date expiration, this will become the current one
+   * The replaceDateExpiration boolean is used in case of replace operation,
```

Comment:
Should be "dateExpirationInPayload" (maybe the change in the name only touched code...)

Reply:
Fixed in 96cd352